### PR TITLE
foreman top menu changes.

### PR DESCRIPTION
**Pull request by abenari**
Do not merge before the corresponding foreman PR get merged!

### CODE REVIEW

**Comment by GregSutcliffe** [comment]
Thanks Amos, I'll keep an eye on it :)

File: lib/foreman_discovery/engine.rb [hunk 1]
```diff
@@ -16,12 +15,36 @@ class Engine < ::Rails::Engine
...
+          permission :destroy_discovered_hosts, {:discovers => [:destroy, :submit_multiple_destroy, :multiple_destroy] }
+
+        end
+
+        # Add a new role called 'Discovery' if it doesn't exist
+        role "Discovery", [:perform_discovery]
+
+        #add menu entry
+        menu :top_menu, :discovery, :url_hash => {:controller=> :discovers, :action=>:index},
+             :caption=> 'Discovered Hosts',
```
Comment (domcleal) [inline lib/foreman_discovery/engine.rb]: N_(), and lower case "H" to fit in with the other menus

**Comment by domcleal** [comment]
:+1: with the comment above.  Obviously, this will lock the branch to 1.4+

**Comment by abenari** [comment]
@GregSutcliffe the PR is ready for merge, all dependencies are merged.

File: lib/foreman_discovery/engine.rb [hunk 2]
```diff
@@ -16,12 +15,36 @@ class Engine < ::Rails::Engine
...
+          permission :edit_discovered_hosts, {:discovers => [:edit, :update, :update_multiple_location,
+                                                             :select_multiple_organization,
+                                                             :update_multiple_organization,
+                                                             :select_multiple_location] }
+          permission :destroy_discovered_hosts, {:discovers => [:destroy, :submit_multiple_destroy, :multiple_destroy] }
+
+        end
+
+        # Add a new role called 'Discovery' if it doesn't exist
+        role "Discovery", [:perform_discovery]
```
Comment (ohadlevy) [inline lib/foreman_discovery/engine.rb]: should this map to the security block above?

**Comment by GregSutcliffe** [comment]
@abenari could you rebase this onto the develop branch? master is intended to track the latest stable release of Foreman, which doesn't have the plugin support yet.

**Comment by abenari** [comment]
replaced by#42 rebased to the correct branch

**Pull request closed without merging**